### ISSUE TITLE
[Screen Reader - Open and close New Tab]: Same name as "Close Tab" is defined for all the close button present at each tab.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8476]
**User Experience:**
Users who rely on screen reader would be impacted and not able to get on which tab close button focus is present as same name "Close Tab" is defined for all the tabs close button.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0
Tool: Accessibility Insights for Windows

**Repro Steps:**
1. Open Windows terminal app.
2. Press "Ctrl+Shift+T" to open multiple tabs.
3. Now check the name defined for close button of each tab using accessibility insight tool.
4. Observe the issue.

**Actual Result:**
**Same name as "Close Tab" is defined for all the close button present at each tab**. So when screen reader focus navigates to any tab close button then narrator announces it as "Close tab button".

**Expected Result:**
**Descriptive and unique name** should be defined for each tab close button like **"Close Tab 1"**.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B248054a6-5e68-4771-9e1e-242fb5025730%7D&cid=e71aec14-4173-423e-aecb-2472bbc751e8

[MAS4.1.2_Close tab button name should be descriptive.zip](https://github.com/microsoft/terminal/files/5635316/MAS4.1.2_Close.tab.button.name.should.be.descriptive.zip)

![MAS4 1 2_Close tab_AI code snippet](https://user-images.githubusercontent.com/53220700/100995527-e9862980-357d-11eb-9d76-9488ba96afcd.png)

**Comment by zadjii-msft** [comment]
Huh, can we even set the automation name of that button at runtime? IIRC it's defined in XAML. That one might actually be defined in the `TabViewItem` itself, I'm not sure we're customizing that at all. This might need to be filed upstream.

**Comment by DHowett** [comment]
![image](https://user-images.githubusercontent.com/189190/101078272-cd24c580-355a-11eb-879e-8cc52dae5a80.png)

This is true of Chromium as well. Why should Terminal be any different?

**Comment by Chandresh1007** [comment]
Feature team disagrees. WinUI2 team disagrees.
The WinUI team thinks that this would be audible clutter. Because you can only get to the Close button using “Items” navigation, it will read the contents of the tab header (“Windows PowerShell”) before you get to “Close Tab”. Therefore, it would say “Windows powershell. Close Tab Windows Powershell”.

Hence, Closing the bug.
